website/project/model.py

```diff
@@ -2158,8 +2159,13 @@ def add_log(self, action, params, auth, foreign_user=None, log_date=None, save=T
             foreign_user=foreign_user,
             params=params,
         )
+
         if log_date:
             log.date = log_date
+
+        self.date_modified = log.date.replace(tzinfo=None)
```

Comment:
Made a change to `def update_node` in `/website/project/views/node.py`. Pulled date_modified from the updated_fields_dict was being returned.     There were two tests in test_views that were failing before this change.

I'm not sure that update_node is even being used?  When you change the title, it is edit_node that is called, not update_node.